<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentWindow</class>
 <widget class="QMainWindow" name="StudentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>431</width>
      <height>301</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btn_student_check">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(220, 219, 255);
color: rgb(0,0,0);

border:0px groove gray;border-radius:
7px;padding:2px 4px;
font: 16pt &quot;Candara&quot;;
</string>
     </property>
     <property name="text">
      <string>查询成绩</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_student_total_score">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>250</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(220, 219, 255);
color: rgb(0,0,0);

border:0px groove gray;border-radius:
7px;padding:2px 4px;
font: 16pt &quot;Candara&quot;;
</string>
     </property>
     <property name="text">
      <string>一分一段表</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_student_revise_password">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(220, 219, 255);
color: rgb(0,0,0);

border:0px groove gray;border-radius:
7px;padding:2px 4px;
font: 16pt &quot;Candara&quot;;

</string>
     </property>
     <property name="text">
      <string>修改密码</string>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="student_label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>欢迎使用高考成绩管理系统学生端</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_student_return">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>460</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;宋体&quot;;
background-color: rgb(204, 204, 191);</string>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
